app/views/dish/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1>dish show</h1>
+
+
+<section id="dish-<%= @dish.id %>">
+  <h2><%= @dish.name %></h2>
+  <p><%= @dish.description %> Total calories: <%= @dish.total_calories %></p>
+</section>
+<section id="ingredients-<%= @dish.id %>">
+  <h3>Ingredients</h3>
+  <ul>
+    <% @dish.ingredients.each do |ingredient| %>
+      <li><%= ingredient.name %></li>
+    <% end %>
+  </ul>
+     <%= render partial: 'form', locals: {
```

Comment:
Nice work using a partial!! For an IC, it's totally fine to not include partials for time but this is great practice!